frontend/app/services/MemberService.scala

```diff
@@ -110,73 +111,85 @@ class MemberService(identityService: IdentityService,
 
-    def createPaidZuoraSubscription(sfContact: ContactId, stripeCustomer: Customer, paid: PaidMemberJoinForm, email: String): Future[String] =
+    def createPaidZuoraSubscription(sfContact: ContactId, paid: PaidMemberJoinForm, email: String, stripeCustomer: Option[Customer]): Future[String] =
       (for {
         zuoraSub <- createPaidSubscription(sfContact, paid, paid.name, paid.tier, stripeCustomer, campaignCode, email, ipAddress, ipCountry)
       } yield zuoraSub.subscriptionName).andThen {
-        case Failure(e: PaymentGatewayError) => logger.warn(s"Could not create paid Zuora subscription due to payment gateway failure: ID=${user.id}; Stripe Customer=${stripeCustomer.id}", e)
-        case Failure(e) => logger.error(s"Could not create paid Zuora subscription: ID=${user.id}; Stripe Customer=${stripeCustomer.id}", e)
+        case Failure(e: PaymentGatewayError) => logger.warn(s"Could not create paid Zuora subscription due to payment gateway failure: ID=${user.id}", e)
+        case Failure(e) => logger.error(s"Could not create paid Zuora subscription: ID=${user.id}", e)
       }
 
     def createFreeZuoraSubscription(sfContact: ContactId, formData: JoinForm, email: String) =
       (for {
         zuoraSub <- createFreeSubscription(sfContact, formData, email, ipAddress, ipCountry)
       } yield zuoraSub.subscriptionName).andThen { case Failure(e) =>
-        logger.error(s"Could not create free Zuora subscription for user ${user.id}", e)}
+        logger.error(s"Could not create free Zuora subscription for user ${user.id}", e)
+      }
 
     def updateSalesforceContactWithMembership(stripeCustomer: Option[Customer]): Future[ContactId] =
       salesforceService.updateMemberStatus(user, tier, stripeCustomer).andThen { case Failure(e) =>
-        logger.error(s"Could not update Salesforce contact with membership status for user ${user.id}", e)}
+        logger.error(s"Could not update Salesforce contact with membership status for user ${user.id}", e)
+      }
 
     formData match {
-      case paid: PaidMemberJoinForm => //
+      case paid@PaidMemberJoinForm(_, _, PaymentForm(_, _, Some(_)), _, _, _, _, _, _, _, _, _) => //Paid member with PayPal token
         for {
-          stripeCustomer  <- createStripeCustomer(paid)
-          idUser          <- getIdentityUserDetails()
-          sfContact       <- createSalesforceContact(idUser)
-          zuoraSubName    <- createPaidZuoraSubscription(sfContact, stripeCustomer, paid, idUser.primaryEmailAddress)
-          _               <- updateSalesforceContactWithMembership(Some(stripeCustomer))  // FIXME: This should go!
-          _               <- updateIdentity()
+          idUser <- getIdentityUserDetails()
+          sfContact <- createSalesforceContact(idUser)
+          zuoraSubName <- createPaidZuoraSubscription(sfContact, paid, idUser.primaryEmailAddress, None)
+          _ <- updateSalesforceContactWithMembership(None) // FIXME: This should go!
```

Comment:
FYI, the FIXME is covered here: https://trello.com/c/6EXHhV6x/99-remove-salesforce-call-to-update-membership-tier-in-contact-object